### DISCUSSION TITLE
v1.0.0-pre1

### DISCUSSION THREAD

**Discussion by tolwi** [tolwi/hassio-ecoflow-cloud#266] [Announcements]
## What's Changed
* Initial RIVER_MINI support by @Bravo13 in https://github.com/tolwi/hassio-ecoflow-cloud/pull/263
* Fixed `AC In Volts` sensor for River Pro by @merlin-zaraza in https://github.com/tolwi/hassio-ecoflow-cloud/pull/259
* public api by @tolwi in https://github.com/tolwi/hassio-ecoflow-cloud/pull/265

## New Contributors
* @Bravo13 made their first contribution in https://github.com/tolwi/hassio-ecoflow-cloud/pull/263
* @merlin-zaraza made their first contribution in https://github.com/tolwi/hassio-ecoflow-cloud/pull/259

**Full Changelog**: https://github.com/tolwi/hassio-ecoflow-cloud/compare/v0.13.4...v1.0.0-pre1

<hr /><em>This discussion was created from the release <a href='https://github.com/tolwi/hassio-ecoflow-cloud/releases/tag/v1.0.0-pre1'>v1.0.0-pre1</a>.</em>

**Comment by merlin-zaraza** [comment]
Hey @tolwi, thanks for release. What are the differences in private vs public API?

**Reply by tolwi** [reply]
"private api" was just reverse-engineered from mobile apps, while public API is the official one - https://developer-eu.ecoflow.com/us/document/introduction
